Docker/PlexAniSync/Dockerfile

```diff
@@ -10,14 +10,30 @@ ENV PLEX_SECTION=Anime \
     ANI_USERNAME='' \
     ANI_TOKEN='' \
     INTERVAL=3600 \
-    PATH="${PATH}:~/.local/bin"
+    SETTINGS_FILE=''
```

Comment:
All the changes to the Docker files and python script should also be applied to the Tautulli ones.